cli/internal/runsummary/task_summary.go

```diff
@@ -80,9 +81,9 @@ type TaskSummary struct {
 	Execution              *TaskExecutionSummary                 `json:"execution,omitempty"` // omit when it's not set
 }
 
-// GetLogs reads the Logfile and returns the data
+// GetLogs reads the log file and returns the data
 func (ts *TaskSummary) GetLogs() []byte {
-	bytes, err := os.ReadFile(ts.LogFile)
+	bytes, err := os.ReadFile(ts.LogFileAbsolutePath.ToString())
```

Comment:
`AbsoluteSystemPath` has a `ReadFile` method so you don't need to convert to a string here